CMakeLists.txt

```diff
@@ -313,6 +313,7 @@ else()
 		set(SUNSHINE_EXECUTABLE_PATH "sunshine")
 	endif()
 	configure_file(gen-deb.in gen-deb @ONLY)
+	configure_file(sunshine.desktop.in sunshine.desktop @ONLY)
```

Comment:
Did I do something incorrectly here? sunshine.desktop is not found during AppImage build.

https://github.com/SunshineStream/Sunshine/runs/5351482081?check_suite_focus=true#step:6:653